Open in browser

stokito opened the pull request:
Most containers expose some http port. We can open it in browser.
For simplicity we can just open first container's published port.
We'll determine the ip:port on container creation and store to meta "Web Port".
To open browser on any platform was added a new dependency github.com/pkg/browser which is very small.

The similar functionality already exists in DockerStation

skanehira commented on gui/keybindings.go:
```diff
@@ -564,6 +565,15 @@ func (g *Gui) commitContainerForm() {
+	if container.WebPort == "" {
+		return
+	}
+	link := "http://" + container.WebPort + "/"
```
I think the last `/` doesn't need

stokito replied:
well, it doesn't hurt but makes the URL complaint with URL spec
If you don't like it feel free to remove it

skanehira reviewed (changes requested):
Thank you :)
I have a comment.

stokito committed:
Open in browser

Most containers expose some http port. We can open it in browser.
For simplicity we can just open first container's published port.
We'll determine the ip:port on container creation and store to meta "Web Port".
To open browser on any platform was added a new dependency github.com/pkg/browser which is very small

stokito commented:
I removed the / in the end and also I removed a check that port is tcp because HTTP3 will be UDP based

skanehira commented:
@stokito Thank you :)
I'll approve this PR when you fix lint error

stokito commented:
sorry but I don't have time anymore, please merge and fix it yourself

stokito commented:
@skanehira please merge when you'll have a minute
